### PR TITLE
Jotai: add useHydrateAtoms docs

### PR DESCRIPTION
**Pull request by Thisen**
Add docs for useHydrateAtoms. It's missing an example, I'll try to add an example to the Next.js using Jotai.

### CODE REVIEW

File: docs/jotai/guides/nextjs.mdx [hunk 1]
```diff
@@ -16,16 +18,3 @@ const postData = atom((get) => {
...
-
-## Examples
-
-### Clock
-
-<Codesandbox id="5ylrj" />
-
-### HN Posts
-
-<Codesandbox id="pm0l8" />
```
Comment (dai-shi) [inline docs/jotai/guides/nextjs.mdx:31]: These examples are good to show. Would you fork and create useHydrateAtoms versions? Or let me know if you want me to work on them.

Comment (Thisen) [reply docs/jotai/guides/nextjs.mdx:31]: We should probably fork them 🙂

**Comment by Thisen** [comment]
@dai-shi Ready for review.

**Review by dai-shi — APPROVED** [review]
Look good. Thanks. Let's  keep improving docs.

**Pull request merged**